dev/reference/tools/sync-diff-inspector.md

```diff
@@ -47,10 +48,17 @@ sample-percent = 100
 # If disabled, data is compared line by line.
 use-checksum = true
 
-# If set to true, data check is ignored. If set false, data is checked.
+# If set to true, data is checked by calculating checksum. Data will not be checked after data is detected, even if the upstream and downstream checksums are inconsistent.
```

Comment:
Strictly speaking, the subject of "If set to true" and "data is ..." must be the same.

```suggestion
# If it is set to true, data is checked only by calculating checksum. Data will not be checked after inspection, even if the upstream and downstream checksums are inconsistent.
```